ceph/samples/ceph-container-source.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: sources.eventing.knative.dev/v1alpha1
+kind: ContainerSource
+metadata:
+  name: ceph-bucket-notification-source
+  labels:
+      app: ceph-source 
+spec:
+  image: docker.io/yuvalif/ceph-bucket-notification-source
```

Comment:
replace the `docker.io` with `knative.dev/eventing-contrib/ceph/cmd/receive_adapter/`